src/amo/reducers/home.js

```diff
@@ -100,6 +100,9 @@ export type ExternalHeroCallToActionType = {|
   text: LocalizedString | null,
 |};
 
+// Type definitions are the same but we have an alias for better readability.
+export type ExternalFooterType = ExternalHeroCallToActionType;
```

Comment:
yeah... so why are we doing this?  We changed the API to use the CTA format so frontend wouldn't need to have two subtly different data structures for essentially the same thing and now we have the exact same data structure here but defined twice. Can't we just rename `ExternalHeroCallToActionType` and `createInternalHeroCallToAction ` to something more generic to still be DRY?

Reply:
Would you have a name to suggest? I couldn't find one...

Reply:
thinking, what it effectively is: `ExternalLinkWithTextType` and `CreateInternalLinkWithText`?  not great, I'll admit.

Reply:
I take that.